src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSliderUI.java

```diff
@@ -150,7 +150,10 @@ public class BasicSliderUI extends SliderUI{
 
     /**
      * Constructs a {@code BasicSliderUI}.
+     * @deprecated This constructor was exposed erroneously and will be removed in next version.
+     *             Use {@link #BasicSliderUI(JSlider b)} instead.
```

Comment:
Did you build the docs and check that the `@link` command works? I would have expected it without the named parameter: `{@link #BasicSliderUI(JSlider)}`.

Also, you might want to change "next version" to "a future version" to allow more flexibility as to the timing of the removal.